compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -138,8 +143,8 @@ createProjectPackageDb opts thisSdkVer deps0 dataDeps = do
     forM_ pkgIdsTopoSorted $ \vertex -> do
         let ((src, templInstSrc, uid, dalf, bs), pkgId, _) =
                 vertexToNode vertex
-        when (uid /= primUnitId) $ do
-            let unitIdStr = unitIdString uid
+        let unitIdStr = unitIdString uid
+        unless (unitIdString primUnitId `isPrefixOf` unitIdStr) $ do
```

Comment:
Why do we need to exclude `daml-prim` here? Isn’t that what resulted in the `daml-prim` package from older SDKs not being contained in the DAR?

Reply:
as discussed, we should probably include in the new upgrading, but we'll wait till we can test this.